apps/fm/js/fm.js

```diff
@@ -619,12 +622,16 @@ var favoritesList = {
     elem.id = this._getUIElemId(item);
     elem.className = 'fav-list-item';
     elem.setAttribute('role', 'option');
-    var html = '';
-    html += '<div class="fav-list-frequency">';
-    html += item.frequency.toFixed(1);
-    html += '</div>';
-    html += '<div class="fav-list-remove-button"></div>';
-    elem.innerHTML = html;
+
+    var subElem = document.createElement('div');
+    subElem.className = 'fav-list-frequency';
+    subElem.textContent = item.frequency.toFixed(1);
+    subElem.dataset.unit = mhzString;
```

Comment:
I can change the string. I'll post a followup.

As for creating here, it is because it is called when you create a bookmark. So it gets called in two cases:
1. when the app starts to show the existing bookmarks
2. when the use create a bookmark.
(here bookmark is a radio frequency you want to keep)

In the DOMLocalized event it just relocalize the one currently displayed.